src/engine/client/backend/vulkan/backend_vulkan.cpp

```diff
@@ -47,7 +47,7 @@
 
 using namespace std::chrono_literals;
 
-class CCommandProcessorFragment_Vulkan : public CCommandProcessorFragment_GLBase
+class CCommandProcessorFragment_Vulkan : public CCommandProcessorFragmentGLBase
```

Comment:
```suggestion
class CCommandProcessorFragmentVulkan : public CCommandProcessorFragmentGlBase
```
I guess the `Vulkan` one should also be renamed.